packages/eslint-plugin-office-addins/src/main.ts

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+    configs: {
+        recommended: {
+            plugins: ['office-addins'],
+            parserOptions: {
+                ecmaFeatures: {
+                    jsx: true
+                  }
+            },
+            extends: ['eslint:recommended'],
+            rules: {}
+        },
+        react: {
+            plugins: ['office-addins'],
+            parserOptions: {
+                ecmaFeatures: {
+                    jsx: true
+                  }
+            },
+            extends: ['plugin:react/recommended'],
+            rules: {}
+        },
+        reactnative: {
+            plugins: ['office-addins'],
+            parserOptions: {
+                ecmaFeatures: {
+                    jsx: true
+                  }
+            },
+            extends: ['plugin:react-native/all'],
+            rules: {}
+        }
+    }
+  };
```

Comment:
It is better to have a newline on the last line. GitHub diff shows the fact that one is missing.